packages/portfolio-contract/src/resolver/resolver.exo.ts

```diff
@@ -94,10 +126,52 @@ export const prepareResolverKit = (
             key,
             harden({ destinationAddress, amountValue, vowKit }),
           );
+          const [chain, chainId, ethAddress] = destinationAddress.split(':');
+          this.facets.vstorage.insertPendingTransaction(
+            amountValue,
+            `${chain}:${chainId}`,
+            ethAddress as `0x${string}`,
+          );
+
           trace(`Registered pending CCTP transaction: ${key}`);
           return vowKit.vow;
         },
       },
+      vstorage: {
+        insertPendingTransaction(
+          amount: bigint,
+          chain: CaipChainId,
+          receiver: `0x${string}`,
+        ) {
+          const value = {
+            type: 'cctp',
```

Comment:
likewise, this is external interface. I suggest hoisting to a constant in `types.ts`. Especially given the interesting choice of lower case.